src/figuresClasses.ts

```diff
@@ -1,19 +1,91 @@
-export interface Figure {
+export enum Shape {
+  TriangleType = 'triangle',
+  CircleType = 'circle',
+  RectangleType = 'rectangle',
```

Comment:
why do you need `Type` everywhere?
```suggestion
  Triangle = 'triangle',
  Circle = 'circle',
  Rectangle = 'rectangle',
```